src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3189,6 +3225,19 @@ static void updateEntity
     currentGeoJson = getObjectFieldF(loc, ENT_LOCATION_COORDS);
   }
 
+  /* Is the entity using date expiration? In that case, we fill the currentdateExpiration attribute with that information.
+   * In any case, if the request contains a new date expiration, this will become the current one
+   * The replaceDate boolean is used in case of replace operation,
```

Comment:
"replaceDate boolean" -> "replaceDateExpiration boolean"

Reply:
Fixed in faea551